knowledge_repo/app/deploy/gunicorn.py

```diff
@@ -21,12 +21,21 @@ def __init__(self, *args, **kwargs):
         BaseApplication.__init__(self)
 
     def load_config(self):
-        options = {
+        args = self.cfg.parser().parse_args(self.cfg.get_cmd_args_from_env())
+
+        # Load up environment configuration.
+        for key, value in vars(args).items():
+            if key != 'args' and value is not None:
```

Comment:
why are we filtering out for keys that are `args` here?

Reply:
`args` here is a config object which contains the `arg` field which should be ignore. This logic is from [here](https://github.com/benoitc/gunicorn/blob/master/gunicorn/app/base.py#L150).